vision.py

```diff
@@ -145,19 +152,33 @@ def create_annotated_display(
 
     def get_vertical_angle(self, p: int):
         """Gets angle of point p above the horizontal.
-        Parameter p should have 0 at the bottom of the frame and FRAME_HEIGHT at the top. """
+        Parameter p should have 0 at the top of the frame and FRAME_HEIGHT at the bottom. """
         return math.atan2(p - FRAME_HEIGHT, FY)
 
     # get_angle and get_distance will be replaced with solve pnp eventually
     def get_horizontal_angle(self, X: float) -> float:
-        return (
-            (X / FRAME_WIDTH) - 0.5
-        ) * MAX_FOV_WIDTH  # 33.18 degrees #gets the angle
-
-    def get_distance(self, Y: float) -> float:
-        target_angle = self.get_vertical_angle(Y)
-        # print(f"Total Angle: {math.degrees(target_angle + GROUND_ANGLE)}")
-        return (TARGET_HEIGHT - CAMERA_HEIGHT) / math.tan(GROUND_ANGLE + target_angle)
+        return -(
+            ((X / FRAME_WIDTH) - 0.5) * MAX_FOV_WIDTH
+        )  # 33.18 degrees #gets the angle
+        # negtive to make the angle same as the robot
+
+    def get_distance(self, y: int, h: int) -> float:
+        # box = cv2.boundingRect(contour)
+        # target_angle = self.get_vertical_angle(box[1]+box[3]/2)
+        target_angle = (
+            (y / FRAME_HEIGHT)
+        ) * MAX_FOV_HEIGHT  # vertical angle of the target from the center of the camera
+        distance = (2.5 - CAMERA_HEIGHT) / math.tan(GROUND_ANGLE + target_angle)
+        distance = self.translate(distance, 1.7, 0.65, 3, 11)
+
+        distance2 = h
+        distance2 = self.translate(distance2, 45, 18, 3, 11)
+
+        combined_dist = (distance2 + distance) / 2
+        ajusted_dist = (
+            5.21 + -1.15 * combined_dist + 0.145 * combined_dist ** 2
+        )  # gotten from google sheets
```

Comment:
Again, what are these adjustments and why do we need them? This needs at least some comments and should use the numbers as constants with descriptive names.